scripts/run_iqtree.py

```diff
@@ -37,3 +40,26 @@ def midpoint_root(infile, outfile):
 file = open(snakemake.output.rooted, 'w')
 file.write(line)
 file.close()
+# -----------------------------------------------------------------------------
+# # change align_variants.aln
+# align_file=open(snakemake.input.alignment, 'r')
+# new_align_file = open(snakemake.params.temp, 'w')
+# for line in align_file:
+#     line=line.strip()
+#     if ">" in line:
+#         for name in og_files:
+#             og_name=name.split()[0]
+#             if '#' in og_name:
+#                 new_name = og_name.replace('#', '_')
+#                 line = line.replace(new_name, og_name)
+#         new_align_file.write(line + '\n')
+#         # new_align_file.write('\n')
+#     else:
+#         new_align_file.write(line + '\n')
+#         # new_align_file.write('\n')
+# new_align_file.close()
+#
+# # os.remove(snakemake.input.alignment)
+# os.rename(snakemake.params.temp ,snakemake.input.alignment)
+# # delete align_variants
+# # rename temp_align_variants to align_variants
```

Comment:
Remove any commented code that's not being run

Reply:
I wrote this because I think it was transphylo won't work if the isolate names had a `#` in the name. Because then align_variants.aln is changed and it reports an error. Should I un-comment it or just delete it?

Reply:
If this fixes the issue with # in sample names then you should uncomment the code.

Looking at the function, it would be better to first check if there are any `#` in the sample names and only run if that's the case, e.g.
```
new_names = []
replacement_needed = False
for name in og_files:
	og_name=name.split()[0]
    if '#' in og_name:
        new_names.append(og_name.replace('#', '_'))
        replacement_needed = True
    else:
    	new_names.append(og_name)
    	
if replacement_needed:
	align_file=open(snakemake.input.alignment, 'r')
	new_align_file = open(snakemake.params.temp, 'w')
	for line in align_file:
		line=line.strip()
		if ">" in line:
	        new_align_file.write(">" + new_names.pop(0) + '\n')
	    else:
	    	new_align_file.write(line + '\n')
	new_align_file.close()
	align_file.close()
	os.rename(snakemake.params.temp ,snakemake.input.alignment)
```